docs/js.rst

```diff
@@ -71,57 +68,57 @@ All of the mentioned 4 examples will produce a valid "action" object that can be
 Events
 ------
 
-Agile UI also offers a great way to associate your actions with certain client-side events. Those
+Agile UI also offers a great way to associate actions with certain client-side events. Those
 events can be triggered by the user or by other JavaScript code. There are several ways to bind
 `$action`.
 
-To execute actions instantly on page load, use `true` as first argument to `js()`::
+To execute actions instantly on page load, use `true` as first argument to :php:meth:`View::js()`::
 
     $view->js(
         true,
-        new jsExpression('alert([])', ['Hello world']
+        new jsExpression('alert([])', ['Hello world'])
     );
 
-You can also combine both forms together::
+You can also combine both forms::
 
     $view->js(true)->hide();
 
-
-Finally, you can specify name of JavaScript event::
+Finally, you can specify the name of the JavaScript event::
 
     $view->js('click')->hide();
 
-Agile UI also provides support for an `on` event binding. This allows you to apply events on
+Agile UI also provides support for an `on` event binding. This allows to apply events on
 multiple elements::
 
-    $buttons = new Buttons();
+    $buttons = $app->add('View');
```

Comment:
You are right Buttons::class does not exist. 😄
May be we can show how to create a button container using Fomantic UI like this:
````
$buttons = View::addTo($app, ['ui' => 'basic buttons']);
````